bikeshare_app/templates/checkin.html

```diff
@@ -0,0 +1,115 @@
+                <div class="inline fields">
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageYes" name="standing" tabindex="0" value="Yes" class="hidden">
+                            <label>Yes</label>
+                        </div>
+                    </div>
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageNo" name="standing" tabindex="0" value="No" class="hidden">
+                            <label>No</label>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="field" name="damageTextArea">
+                    <label>Explanation:</label>
+                    <textarea></textarea>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="lock icon"></i>
+                Was the bike locked up?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>No</label>
+                    </div>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="privacy icon"></i>
+                Was the key returned?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>No</label>
+                    </div>
+                </div>
+            </div>
+            <div class="ui fluid large bingColor submit button">Submit</div>
+
+            <div class="ui error message"></div>
+        </form>
+
+    </div>
+</div>
+{% endblock %}
+
+{% block additionalScripts %}
+<script>
+    $('div[name="damageTextArea"]').hide();
+    $('.ui.radio.checkbox').checkbox();
+
+    $('input[id="damageYes"]').change(function() {
+        $('div[name="damageTextArea"]').show();
+    });
+
+    $('input[id="damageNo"]').change(function() {
+        $('div[name="damageTextArea"]').hide();;
```

Comment:
Excuse me there is two semicolons on this line where do you think we get them you think they just grow on trees. I'll have you know I mastered programming within hours. I wrote my OWN versions of the two most popular programs in the world.
The first is Hello World. While rudimentary, and lacking in threading capabilities, it boasts an extremely user-friendly and intuitive interface. I wrote it in c++ using gcc compiler, Asus monitor, and human fingers.
The second is Good Bye world. While it may not be as widely known as Hello World, it still offers the EXACT same amount of utility that the hello world program offers. I wrote this in Java using some coffee grounds I found at the bottom of my Starbucks cup. I compiled it myself with a straw.
I too am thinking of expanding on my Hello World program and turning it into a multi-greeting, synchronized planetary operating system based off the not widely known Iinux kernel (which I also made by hand using binary found in the background of the Matrix (the first), specifically minutes 22 thru 34 as a base.) I've yet to compile my binary into assembly, and then compile the assembly into c++ so that I can finally recompile into an exe, but when I do... I'll be rich! I'm going to name the OS Look Out World, or LOW for short. So look out world, there will be a new LOW coming from me soon!!